app/core/less/variables.less

```diff
@@ -26,6 +26,7 @@
 @purple-highcontrast:    #291A45;
 @purple-success:         #7950ea;
 @purple-blip:            #4E4490;
+@purple-uploader:        #291448;
```

Comment:
Yes, exactly. I'll be discussing this with Sara during our design review. I'll be sure to loop you all in after.